# note: yarn patch-package libp2p/it-byte-stream  to patch nested dep
diff --git a/node_modules/libp2p/node_modules/it-byte-stream/dist/src/index.js b/node_modules/libp2p/node_modules/it-byte-stream/dist/src/index.js
index 5ec0fb0..9b721cf 100644
--- a/node_modules/libp2p/node_modules/it-byte-stream/dist/src/index.js
+++ b/node_modules/libp2p/node_modules/it-byte-stream/dist/src/index.js
@@ -69,7 +69,7 @@ export function byteStream(duplex, opts) {
             let listener;
             const abortPromise = new Promise((resolve, reject) => {
                 listener = () => {
-                    reject(new AbortError('Read aborted'));
+                    // reject(new AbortError('Read aborted'));
                 };
                 options?.signal?.addEventListener('abort', listener);
             });
